app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>SampleApp</title>
```

Comment:
- Research about `Rails I18n` and apply for project
refs: https://guides.rubyonrails.org/i18n.html#setup-the-rails-application-for-internationalization
- Default setting 2 languages: English - en, Japanese - jp